### PR TITLE
Enable https

### PR DESCRIPTION
**Pull request by hughsimpson**
I'm speaking to a jaeger server on https, and the hardcoded `http` url blocks that. This works for me, so sharing.

### CODE REVIEW

**Comment by dpsoft** [comment]
Hey @hughsimpson, please remember to sign the [CLA](https://docs.google.com/forms/d/e/1FAIpQLSdhLT5zYRzdzRKRybFJcXUUuhpdJzRiiDhPZKtCTgToQaqpGw/viewform) to merge this PR :) Thanks a lot

**Comment by hughsimpson** [comment]
NP

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
-version in ThisBuild := "1.0.2-SNAPSHOT"
+version in ThisBuild := "1.0.3-SNAPSHOT"
```
Comment (ivantopo) [inline version.sbt]: Hey @hughsimpson, maybe you ran a local release or something? this file should be showing 1.0.2-SNAPSHOT :)

**Review by ivantopo — CHANGES_REQUESTED** [review]
Thanks @hughsimpson, please just revert the changes to `version.sbt` and we are good to go :smiley:

**Comment by hughsimpson** [comment]
@ivantopo should be addressed now, thanks

**Comment by dpsoft** [comment]
@hughsimpson thanks for proposing this PR!

**Comment by dpsoft** [comment]
@hughsimpson https://github.com/kamon-io/kamon-jaeger/releases/tag/v1.0.2
published on our bintray repo and on its way to Maven Central

**Pull request merged**